apiary.apib

```diff
@@ -1700,6 +1700,130 @@ root will not be listable, as it is virtual. Access known paths directly.
 ## gc
 
 # Group resolve
+Resolve the value of names to IPFS.
+
+## resolve [GET /resolve{?arg}{&resolve}]
+There are a number of mutable name protocols that can link among
+themselves and into IPNS.  For example IPNS references can (currently)
+point at IPFS object, and DNS links can point at other DNS links, IPNS
+entries, or IPFS objects. This command accepts any of these
+identifiers and resolves them to the referenced item.
+
++ Parameters
+    + arg (string, required) - The name to resolve.
+    + recursive (boolean, optional) - Resolve until the result is an IPFS name
```

Comment:
No idea. The Wireshark HTTP request doesn't have a flag unless provided. @diasdavid?